lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -195,7 +203,8 @@ def instructor_dashboard_2(request, course_id):
 
     if can_see_special_exams:
         sections.append(_section_special_exams(course, access))
-
+    if course_id == 'course-v1:HarvardX+CS50+X':
+        log.info('Investigating log at %s : section certificate', time.time() - start_time)
```

Comment:
We can initiate a new time object here instead. That object can be used as the reference from here on. It will make the logs more clean

Reply:
I've added some changes. I've put it before calling `_section_student_admin` since this method further calls `_is_small_course`